docs/node/workers.md

```diff
@@ -0,0 +1,51 @@
+---
+id: workers
+title: Workers in Node
+sidebar_label: Workers
+---
+
+## What is a Worker?
+
+A Worker is a service that executes [Workflows](/docs/go/workflows) and [Activities](/docs/go/activities).
```

Comment:
Link to the relevant `node` pages instead of `go`.
Also is the term service accurate here? Maybe call it a process?